src-plugins/manualRegistration/manualRegistration.h

```diff
@@ -92,6 +91,10 @@ class MANUALREGISTRATIONPLUGIN_EXPORT manualRegistration : public itkProcessRegi
     */
     virtual bool writeTransform(const QString& file);
 
+    void setParameter(int data, int channel);
```

Comment:
Even better, use the most appropriate overload of ```setParameter```. There are many versions, I don't understand why it's always the same one being used. Here, since there is only one parameter to set, then the non-channel version is more suited:

```
    virtual void setParameter(int data);
    virtual void setParameter(int data, int channel);
    virtual void setParameter(int data, int channel, int frame);

    virtual void setParameter(int *data);
    virtual void setParameter(int *data, int channel);
    virtual void setParameter(int *data, int channel, int frame);

    virtual void setParameter(qlonglong data);
    virtual void setParameter(qlonglong data, int channel);
    virtual void setParameter(qlonglong data, int channel, int frame);

    virtual void setParameter(qlonglong *data);
    virtual void setParameter(qlonglong *data, int channel);
    virtual void setParameter(qlonglong *data, int channel, int frame);

    virtual void setParameter(double  data);
    virtual void setParameter(double  data, int channel);
    virtual void setParameter(double  data, int channel, int frame);

    virtual void setParameter(double *data, int count);
    virtual void setParameter(double *data, int count, int channel);
    virtual void setParameter(double *data, int count, int channel, int frame);

    virtual void setParameter(dtkAbstractObject*  data);
    virtual void setParameter(dtkAbstractObject*  data, int channel);
    virtual void setParameter(dtkAbstractObject*  data, int channel, int frame);
```